smartsim/settings/launch_settings.py

```diff
@@ -126,6 +126,18 @@ def __init__(
             """The launcher type"""
```

Comment:
In the modules arguments/launch/... there is a spelling error where Overwriting is written as Overwritting - could you fix this in this PR?

Reply:
cant find this error